cmp/grid/Grid.ts

```diff
@@ -201,10 +213,10 @@ class GridLocalModel extends HoistModel {
                 clipboardCopy: Icon.copy({asHtml: true})
             },
             components: {
-                agColumnHeader: props => columnHeader({gridModel: model, ...props}),
-                agColumnGroupHeader: props => columnGroupHeader(props)
+                agColumnHeader: columnHeader,
+                agColumnGroupHeader: columnGroupHeader
             },
-            rowSelection: selModel.mode,
+            rowSelection: selModel.mode == 'disabled' ? undefined : selModel.mode,
```

Comment:
Makes sense-  Do we need to advertise this as a 'breaking change'?

Reply:
was not intended to be a breaking change!  Will test this more carefully -- my assumption was that this was just a change to satisfy typescript